samples/constitutions/test/resolve.js

```diff
@@ -1,3 +1,43 @@
+function getMemberInfo(memberId) {
+  return ccf.bufToJsonCompatible(
+    ccf.kv["public:ccf.gov.members.info"].get(ccf.strToBuf(memberId))
+  );
+}
+
+function isRecoveryMember(memberId) {
+  return (
+    ccf.kv["public:ccf.gov.members.encryption_public_keys"].get(
+      ccf.strToBuf(memberId)
+    ) ?? false
+  );
+}
+
+// Defines which of the members are operators.
+function isOperator(memberId) {
+  // Operators cannot be recovery members.
+  if (isRecoveryMember(memberId)) {
+    return false;
+  }
+  return getMemberInfo(memberId).member_data?.is_operator ?? false;
+}
+
+// Defines which of the members are operator provisioners.
+function isOperatorProvisioner(memberId) {
+  return getMemberInfo(memberId).member_data?.is_operator_provisioner ?? false;
+}
+
+// Defines actions that can be passed with sole operator provisioner input.
+function canOperatorProvisionerPass(action) {
+  // Operator provisioners can add or retire operators.
+  return (
+    {
+      set_member_data: () => action.args["member_data"]?.is_operator ?? false,
```

Comment:
This is different from the other `resolve.js` file - is that on purpose?

Reply:
I originally thought it was okay to include but it wasn't in the end, so removed in https://github.com/microsoft/CCF/pull/4343/commits/5223abe0d0d4449667145610fae7762f9daf688b